ArduPlane/Parameters.h

```diff
@@ -401,6 +404,9 @@ class Parameters {
     AP_Int16 waypoint_radius;
     AP_Int16 waypoint_max_radius;
     AP_Int16 rtl_radius;
+    
+    // Thermalling
+    AP_Int16 loiter_radius;
```

Comment:
Oh, got it. Can I rename this "local" parameter into something like "thermalling_radius" then? I do think we need both a variable for the "normal" loiter radius (aparm.loiter_radius) and for the "thermalling" loiter radius (e.g., g.thermalling_radius)